TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -599,6 +599,18 @@ TYPO3:
             namespaces:
               f: 'TYPO3\Fluid\ViewHelpers'
               neos: 'TYPO3\Neos\ViewHelpers'
+      'Neos:TypoScriptViewHelpers':
```

Comment:
Yes this is intended. Currently the view ts:render view helper is simply missing in the docs.